### DISCUSSION TITLE
Custom field URL needs a rel parameter

### DISCUSSION THREAD

**Discussion by helgator** [joomla/joomla-cms#29058] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
external links need to have the rel attribute noopener or noreferrer

### Describe the solution you'd like
* add parameter field to add multiple rel attributes separated by space
* admin must be able to set default value
* this parameter needs its own permission so it can be set to allow only certain users to change its value

### Additional context
none

**Comment by Quy** [comment]
Related #24337

**Comment by helgator** [comment]
Related indeed but I would prefer a user/admin controlled parameter and not a system wide general behavior because there are more rel attributes than just noopener or noreferrer. If #24337 becomes a parameter to the Custom Field URL my issue can be closed.
